docs/esa/neocc.rst

```diff
@@ -0,0 +1,327 @@
+ESA NEOCC Portal Python Interface Library (`astroquery.esa.neocc`/`astroquery.solarsystem.neocc`)
+************************************************************************************************
+
+The ESA NEO Coordination Centre (NEOCC) is the operational centre of ESA’s Planetary Defence Office
+PDO) within the Space Safety Programme (S2P). Its aim is to coordinate and contribute to the
+observation of small Solar System bodies in order to evaluate and monitor the threat coming from 
+near-Earth objects (NEOs).
+
+ESA NEOCC Portal Python interface library makes the data that `ESA NEOCC <https://neo.ssa.esa.int/>`_
+provides easily accessible through a Python program.
+
+The main functionality of this library is to allow a programmer to easily retrieve:
+
+* All the NEAs
+* Other data that the NEOCC provides (risk list, close approach list, etc.)
+* All basic and advanced information regarding a NEA
+* An ephemeris service for NEAs
+
+==============================
+Getting ESA NEOCC's products
+==============================
+
+--------------------------------
+1. Direct download of list files
+--------------------------------
+This function allows the user to download the requested list data from ESA NEOCC.
+Different lists that can be requested are:
+
+* All NEA list: *nea_list*
+* Catalogue of NEAs (current date): *neo_catalogue_current*
+* Catalogue of NEAs (middle arc): *neo_catalogue_middle*
+* Updated NEA list: *updated_nea*
+* Monthly computation date: *monthly_update*
+* Risk list (normal): *risk_list*
+* Risk list (special): *risk_list_special*
+* Close approaches (upcoming): *close_approaches_upcoming*
+* Close approaches (recent): *close_approaches_recent*
+* Priority list (normal): *priority_list*
+* Priority list (faint): *priority_list_faint*
+* Close encounter list: *close_encounter*
+* Impacted objects: *impacted_objects*
+ 
+These lists are referenced in `<https://neo.ssa.esa.int/computer-access>`_.
+
+--------------------------------
+Examples
+--------------------------------
+**NEA list:** The output
+of this list is a `astropy.table.Table` which contains the list of all NEAs
+currently considered in the NEOCC system.
+
+.. doctest-remote-data::
+
+    >>> from astroquery.esa.neocc import neocc
+    >>> list_data = neocc.query_list(list_name='nea_list')
+    >>> print(list_data)  # doctest: +IGNORE_OUTPUT
+        NEA     
+    ------------
+        433 Eros
+      719 Albert
+      887 Alinda
+    1036 Ganymed
+       1221 Amor
+             ...
+          2023RJ
+          2023RK
+          2023RL
+          2023RM
+          6344P-L
+    Length = 32564 rows
+
+
+Each asteroid can be accessed using its index. This information can
+be used as input for *query_object* method.
+
+.. doctest-remote-data::
+
+    >>> print(list_data["NEA"][4])
+    1221 Amor
+
+**Close approaches:**  The output of this list is a `astropy.table.Table` which
```

Comment:
I would suggest using headings instead? But have to double check how it all looks like in the rendered version, maybe it's all OK as is.

Reply:
Yes, maybe is better to use headings